assets/github/pipeline.tmpl

```diff
@@ -0,0 +1,23 @@
+name: Pipeline Action
```

Comment:
@vinicius-marinho-nearform we can probably come up with a better name for this 😄

Reply:
You are right hahahaha. Any ideas?

Reply:
Name changed